open_dread_rando/static_fixes.py

```diff
@@ -141,13 +144,11 @@ def _apply_boss_cutscene_fixes(editor: PatcherEditor, cutscene_ref: dict, callba
         })
```

Comment:
```py
def _apply_boss_cutscene_fixes(editor: PatcherEditor, cutscene_ref: dict, callback: str, insert_callback_at: Optional[int] = None)):
    cutscene_player = editor.resolve_actor_reference(cutscene_ref)
    callbacks_after_cutscene = cutscene_player.pComponents.CUTSCENE.vctOnAfterCutsceneEndsLA

    # Add the custom call when the boss dies
    if callback:
        callback_action = {
            "@type": "CLuaCallsLogicAction",
            "sCallbackEntityName": "",
            "sCallback": callback,
            "bCallbackEntity": False,
            "bCallbackPersistent": False,
        }
        if insert_callback_at is None:
        	callbacks_after_cutscene.append(callback_action)
        else:
        i = insert_callback_at	callbacks_after_cutscene[i:i] = [callback_action]
```
if you do it like this you can pass -1 as the index for kraid

Reply:
Ok. I suggest just squash commit and use the first commit message when merging.